tfx/examples/imdb/imdb_utils_native_keras.py

```diff
@@ -27,44 +27,50 @@
 
 from tfx.components.trainer.executor import TrainerFnArgs
 
+""" 
+There are 50,000 entries in the imdb dataset. ExampleGen splits the dataset
+with a 2:1 train-eval ratio.
+Batch_size is an empirically sound configuration
+"""
 _TRAIN_BATCH_SIZE = 64 
 _TRAIN_DATA_SIZE = int(50000 * 2 / 3) 
 _EVAL_BATCH_SIZE = 64
 _EVAL_DATA_SIZE = int(50000 / 3)
 _LABEL_KEY = "sentiment"
-DELIMITERS = '.,!?() '
+_DELIMITERS = '.,!?() '
 _MAX_FEATURES = 8000
 _MAX_LEN = 100
+_LEARNING_RATE = 1e-4
+_TRAIN_EPOCHS = 10
+_DROPOUT_RATE = 0.2
+_HIDDEN_UNITS = 64
 
 def _gzip_reader_fn(filenames):
   """Small utility returning a record reader that can read gzip'ed files."""
   return tf.data.TFRecordDataset(filenames, compression_type='GZIP')
 
-def _sentiment_to_int(sentiment):
-  """Cast the label type to int64"""
-  ints = tf.cast(sentiment, tf.int64)
-  return ints
-
 def _tokenize_review(review):
   """Tokenize the reivews by spliting the reviews, then constructing a
   vocabulary. Map the words to their frequency index in the vocabulary"""
   review_sparse = tf.compat.v1.string_split(
```

Comment:
As discussed, lets se if we can get tf.strings.split or tf.strings.unicode.split to work here.

Reply:
tf.strings.unicode_split splits the text into individual characters. It doesn't take in any separator. So I don't think it'd work
I think there is a potential bug here worth discovering